src/awkward/_typing.py

```diff
@@ -53,7 +53,7 @@
     )
 
 
-JSONSerializable: TypeAlias = (
-    "str | int | float | bool | None | list | tuple | JSONMapping"
-)
+JSONSerializable: (
+    TypeAlias
+) = "str | int | float | bool | None | list | tuple | JSONMapping"
```

Comment:
FYI, next release of Ruff should fix this, and it sounds like it's coming out pretty soon (maybe todayish), so we could wait.